README.md

```diff
@@ -41,7 +41,7 @@ for all `Path` objects (even local ones).
 ### Using this package as a provider for Java 9 and above
 
 With the introduction of modules in Java 9 the extension mechanism was retired. Providers should now be supplied as java modules.
-For backward compatability we have not yet made this change so to ensure that the provider in this package is recognized
+For backward compatibility we have not yet made this change so to ensure that the provider in this package is recognized
```

Comment:
Now that we have moved to JVM 11 we could (probably should?) package as a module?